frontend/src/layout/LatestVersion.js

```diff
@@ -4,7 +4,7 @@ import api from './../lib/api'
 import { Button } from 'antd'
 import { ChangelogModal } from '~/layout/ChangelogModal'
 import { userLogic } from 'scenes/userLogic'
-import { CheckOutlined, WarningOutlined } from '@ant-design/icons'
+import { CheckOutlined, BulbOutlined } from '@ant-design/icons'
```

Comment:
These changes were already merged in #1216

Reply:
Yup, they were, I rebased to work on master at the time and this got in. Shouldn't be a conflict though.